app/css/components/copy-text-to-clipboard.css

```diff
@@ -1,5 +1,8 @@
 @import "../styles";
 
+.c-react-wrapper-common-copy-to-clipboard-button {
+    min-height: 58px;
+}
```

Comment:
@kntsoriano @ErikSchierboom FYI, I'm adding these so that the page doesn't jump around when the react components render. This class automatically gets added to the divs with the react data now.

Reply:
👍